### PR TITLE
chore: update type for view

### PR DESCRIPTION
**Pull request by SwaySway**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/codegen-ui/lib/types/view/view.ts [hunk 1]
```diff
@@ -18,14 +18,16 @@ import { ViewStyle } from './style';
 import { ColumnsMap } from './table';
 
 export interface StudioView {
-  appId?: string;
```
Comment (hein-j) [inline packages/codegen-ui/lib/types/view/view.ts:21]: One thing to note is that this interface is used to instantiate StudioView model in UI.

Comment (hein-j) [reply packages/codegen-ui/lib/types/view/view.ts:21]: I don't think it'll break UI but just a note

**Review by hein-j — APPROVED** [review]

File: packages/codegen-ui/lib/types/view/view.ts (hunk 1 above)
Comment (hein-j) [reply packages/codegen-ui/lib/types/view/view.ts:21]: Maybe sync w/ Kevin

**Comment by SwaySway** [comment]
closing this pr as the original implementation has changed for view types

**Pull request closed without merging**